src/Tasks/Common/Resources/Strings.resx

```diff
@@ -560,4 +560,12 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1113: Failed to create apphost (attempt {0} out of {1}): {2}</value>
     <comment>{StrBegin="NETSDK1113: "}</comment>
   </data>
-</root>
\ No newline at end of file
+  <data name="CannotFindIjwhost" xml:space="preserve">
+    <value>NETSDK1114: Unable to find a .NET Core IJW host. The .NET Core IJW host is only available on .NET Core 3.0 or higher when targeting Windows.</value>
+    <comment>{StrBegin="NETSDK1114: "}</comment>
+  </data>
+  <data name="NETFrameworkWithoutUsingNETSdkDefaults" xml:space="preserve">
+    <value>NETSDK1115: The current .NET SDK does not support .NET Framework below 5 without using .NET SDK Defaults. It is likely due to a mismatch between C++/CLI project CLRSupport property and TargetFramework.</value>
```

Comment:
I am not sure the error is good enough. The direct reason is UsingNETSdkDefaults=false + using full framework. However, from the user point of view. CLRSupport =netcore + targetframework=net4XX is the problem

Reply:
I would remove the "below 5".

Reply:
.NET 5 is not .NET Framework 5

Reply:
Removed "below 5"